### ISSUE TITLE
Update VisionFive 2 Quick Start Guide

### ISSUE THREAD

**Issue by tylerferrara** [starfive-tech/VisionFive2#40]
I found a correction in the Quick Start Guide.
Where can I update this PDF?

---
During boot of OpenSBI, it's important to disable "Hardware Flow Control" and enable "Software Flow Control" so the developer can send keyboard input to the device. Without this, you'll waste time staring at the login prompt, unsure if your keyboard is communicating with the board at all.

**Comment by okias** [comment]
Yes! Yes, please. Already burned few hours before I saw the mention from #13
